python/ray/serve/router.py

```diff
@@ -202,7 +202,10 @@ async def enqueue_request(self, request_meta, *request_args,
 
         # Note: a future change can be to directly return the ObjectID from
         # replica task submission
-        result = await query.async_future
+        try:
+            result = await query.async_future
+        except RayTaskError as e:
+            result = e
```

Comment:
Should we count these too? Can be aggregated from others of course but might be nice to have a first-class global counter.